src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.query.ts

```diff
@@ -29,7 +31,22 @@ export const pupilLessonOverviewQuery =
       });
     }
 
+    const isLegacyLicense =
+      !lesson.lessonCohort || lesson.lessonCohort === LEGACY_COHORT;
+    let transcriptSentences: string | string[] =
+      lesson.transcriptSentences ?? [];
+
+    if (lesson.videoTitle && !isLegacyLicense) {
```

Comment:
Pushing this implementation detail as close to the network as possible. Will make it easier to remove should captions come from GQL later on.